apollo-router/src/query_planner/mod.rs

```diff
@@ -384,10 +384,10 @@ pub(crate) mod fetch {
                             if let Ok(Some(value)) = select_object(content, requires, schema) {
                                 match values.get_index_of(&value) {
                                     Some(index) => {
-                                        paths.insert(path, index);
+                                        paths.insert(path.clone(), index);
```

Comment:
not sure why the clones are required Oo

Reply:
Because the path returned by path and value selection is a reference